CMakeLists.txt

```diff
@@ -259,6 +259,12 @@ if(WarpX_PSATD)
     if(WarpX_DIMS STREQUAL RZ)
         target_link_libraries(ablastr PUBLIC blaspp)
         target_link_libraries(ablastr PUBLIC lapackpp)
+
+        # BLAS++ forgets to declare cuBLAS and cuRAND dependencies
```

Comment:
Did you mean
```suggestion
        # BLAS++ forgets to declare cuBLAS and cudaRT dependencies
```

Reply:
ah yes, I updated this in a follow-up commit but forgot to update the comment. Thx!